rules/CodingStyle/Rector/ClassMethod/OrderAttributesRector.php

```diff
@@ -103,18 +118,70 @@ public function refactor(Node $node): ?Node
     /**
      * @param mixed[] $configuration
      */
-    public function configure(array $configuration): void
+    public function configure(array $configuration = [self::ALPHABETICALLY]): void
     {
         Assert::allString($configuration);
+        Assert::minCount($configuration, 1);
 
-        $this->attributesOrderByName = array_flip($configuration);
+        if ($this->isAlphabetically($configuration)) {
+            $this->configuration = $configuration;
+        } else {
+            $this->configuration = array_flip($configuration);
+        }
+    }
+
+    /**
+     * @param array<AttributeGroup> $originalAttrGroups
+     * @return array<AttributeGroup>
+     */
+    private function sortAlphabetically(array $originalAttrGroups): array
+    {
+        usort($originalAttrGroups, function (
+            AttributeGroup $firstAttributeGroup,
+            AttributeGroup $secondAttributeGroup,
+        ): int {
+            $currentNamespace = $this->getName($firstAttributeGroup->attrs[0]->name);
+            $nextNamespace = $this->getName($secondAttributeGroup->attrs[0]->name);
+            return $this->strcmp($currentNamespace, $nextNamespace);
```

Comment:
I had to do that otherwise I had this error `Use separate function calls with readable variable names`
https://github.com/symplify/phpstan-rules/blob/main/docs/rules_overview.md#nonestedfunccallrule

Reply:
This should only report 2+ nested function calls, not one. That's a bug in the rule and you can ignore it in `phpstan.neon`

Reply:
@TomasVotruba done